src/openrct2/peep/Peep.cpp

```diff
@@ -338,6 +338,21 @@ Peep* rct_sprite::AsPeep()
     return result;
 }
 
+bool rct_sprite::IsVehicle() const
+{
+    return vehicle.sprite_identifier == SPRITE_IDENTIFIER_VEHICLE;
+}
+
+Vehicle* rct_sprite::AsVehicle()
+{
+    Vehicle* result = nullptr;
+    if (IsVehicle())
+    {
+        return reinterpret_cast<Vehicle*>(this);
+    }
+    return result;
+}
+
```

Comment:
It looks so weird to have these on `Peep`, is this for the mini golf?

Reply:
Yes this code should have been put in vehicle.cpp.